packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -271,16 +274,21 @@ const nextServerlessLoader: loader.Loader = function() {
           `
             : `const nowParams = null;`
         }
+        renderOpts.params = _params || params
```

Comment:
What is this line doing? Can we add a comment on why we set it and why there's two values we're oring between?

Reply:
Add comment explaining why we're using `_params` and falling back to `params`